tools/peephole-replace.py

```diff
@@ -100,6 +103,25 @@ def flags_from_enum(enum_value):
     """
     return str(enum_value).replace(f"{type(enum_value).__name__}.", '')
 
+
+def read_enum(filename, prefix):
+    result = {}
+    basepath = os.path.dirname(__file__)
+    for line in open(os.path.join(basepath, "..", "src", filename), "rt"):
+        line = line.strip()
+        if ';' in line:
+            line = line[:line.find(';')]
+        if line.startswith(prefix):
+            if " equ " in line:
+                line = line.split(" equ ")
+                value = line[0].strip()
+                key = line[1].strip()
+                if key.startswith('$'):
+                    key = int(key[1:], 16)
+                    result[key] = value
+    return result
```

Comment:
Nice to read from the enum (rather than duplicating the enum in the peephole script)

Reply:
Not sure why I duplicated it initially, this is much simpler :)